test/game_test.exs

```diff
@@ -4,33 +4,42 @@ defmodule GameTest do
     alias TicTacToe.Board, as: Board
 
     test "it returns starting game struct" do
-        assert Game.setup_new_game == %Game{game_status: :in_progress, 
-                                            player_one: "x",
-                                            player_two: "o",
-                                            current_player: :player_one, 
+        assert Game.setup_new_game == %Game{game_status: "in progress", 
+                                            current_player: "X", 
+                                            incoming_move: nil,
                                             board: Board.empty_board}
     end
     
     describe "updating board in game struct: " do
         test "make move updates game struct for first time" do
-            game = Game.setup_new_game
+            game = %{game_status: "in progress", 
```

Comment:
I suggest reading through the Elixir style guide: https://github.com/christopheradams/elixir_style_guide

Generally, it's preferred to write as: 
```
%Game{
  game_status: "in progress",
  current_player: "X",
  incoming_move: nil,
  board: Board.empty_board
}
```
with the braces on their own separate lines from the contents.

Same principle applies for other braces that span multiple lines, whether they be `{`, `(` or `[`.